nnpdf_data/nnpdf_data/commondata/CMS_1JET_8TEV/metadata.yaml

```diff
@@ -68,7 +68,13 @@ implemented_observables:
       theory: *id002
   theory:
     FK_tables:
-    - - CMS_1JET_8TEV-CMS_1JET_8TEV
```

Comment:
I think in this one we need (like in the one above) the separated `legacy` `legacy_theory`, etc.